### PR TITLE
[SeveralApplications] Comment/remove unused variables 

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**
These variables were triggering warnings due to be unused. I do not completely remove as it is not my code, but most of it looks like leftover from debugging. If developers agree to remove, it will be OK

**🆕 Changelog**
- These variables were triggering warnings due to be unused

### CODE REVIEW

**Review by rubenzorrilla — COMMENTED** [review]
Fine from my side.

**Comment by philbucher** [comment]
what are the warnings?
I am curious, as this should be prevented by the CI
Also I don't get them with any compiler

**Comment by loumalouomega** [comment]
> what are the warnings? I am curious, as this should be prevented by the CI Also I don't get them with any compiler

Declared but unused variable. As I use Manjaro (Arch derived distro), I use the last gcc version and maybe that's why it only appears to me.

File: applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h [hunk 1]
```diff
@@ -1380,7 +1380,7 @@ namespace Kratos
 		//we start with the first position, then it will enter the loop.
 		position = pparticle.Coordinates(); //initial coordinates
 
-		double only_integral  = 0.0 ;
+		//double only_integral  = 0.0 ;
```
Comment (miguelmaso) [inline applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h]:
```suggestion
```

File: applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h [hunk 2]
```diff
@@ -1400,7 +1400,7 @@ namespace Kratos
 				nsubsteps=1;
 			substep_dt = delta_t / double(nsubsteps);
 
-			only_integral = 1.0;// weight;//*double(nsubsteps);
+			//only_integral = 1.0;// weight;//*double(nsubsteps);
```
Comment (miguelmaso) [inline applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h]:
```suggestion
```

File: applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h [hunk 3]
```diff
@@ -1423,7 +1423,7 @@ namespace Kratos
 						noalias(vel) += geom[j].FastGetSolutionStepValue(mVelocityVar)*N[j];
 					}
 
-					only_integral += 1.0; //values saved for the current time step
+					//only_integral += 1.0; //values saved for the current time step
```
Comment (miguelmaso) [inline applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h]:
```suggestion
```

File: applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h [hunk 4]
```diff
@@ -1507,7 +1507,7 @@ namespace Kratos
 		//we start with the first position, then it will enter the loop.
 		position = pparticle.Coordinates(); // + (pparticle)->FastGetSolutionStepValue(DISPLACEMENT); //initial coordinates
 
-		double only_integral  = 0.0 ;
+		//double only_integral  = 0.0 ;
```
Comment (miguelmaso) [inline applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h]:
```suggestion
```

File: applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h [hunk 5]
```diff
@@ -1528,7 +1528,7 @@ namespace Kratos
 				nsubsteps=1;
 			substep_dt = delta_t / double(nsubsteps);
 
-			only_integral = 1.0;// weight;//*double(nsubsteps);
+			//only_integral = 1.0;// weight;//*double(nsubsteps);
```
Comment (miguelmaso) [inline applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h]:
```suggestion
```

File: applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h [hunk 6]
```diff
@@ -1549,7 +1549,7 @@ namespace Kratos
 					}
 
 
-					only_integral += 1.0;//weight ; //values saved for the current time step
+					//only_integral += 1.0;//weight ; //values saved for the current time step
```
Comment (miguelmaso) [inline applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h]:
```suggestion
```

**Review by miguelmaso — COMMENTED** [review]
I used this utility some time ago. Probably more code is unused...

**Review by miguelmaso — APPROVED** [review]

**Pull request merged**